packages/web3/test/e2e/fixtures/sepolia.ts

```diff
@@ -14,6 +14,7 @@ GNU Lesser General Public License for more details.
 You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
+export const sepoliaAddress = '0xedfd52255571b4a9a9d4445989e39f5c14ff0447';
```

Comment:
Maybe we move this to the [.secrets template](https://github.com/web3/web3.js/blob/e550a9c88edec819a8ac0181fc2d321f8e959121/templates/.secrets.json.tmpl#L6) like so:

```json
{
    "SEPOLIA": {
        "HTTP": "",
        "WS": "",
        "ACCOUNT": {
            "address": "0xa127C5E6a7E3600Ac34A9a9928E52521677e7211",
            "privateKey": ""
        },
        "ALLOWED_SEND_TRANSACTION": false,
        "DEPLOYED_TEST_CONTRACT_ADDRESS": "0xedfd52255571b4a9a9d4445989e39f5c14ff0447"
    },
    "MAINNET": {
        "HTTP": "",
        "WS": "",
        "ACCOUNT": {
            "address": "0x98AF911164f9d4E0f5983ed114949c3Bfe3ADc9d",
            "privateKey": ""
        },
        "ALLOWED_SEND_TRANSACTION": false,
        "DEPLOYED_TEST_CONTRACT_ADDRESS": ""
    }
}
```

And add a util function [here](https://github.com/web3/web3.js/blob/4.x/packages/web3/test/e2e/e2e_utils.ts) called `getE2ETestContractAddress`